nautilus_trader/common/actor.pyx

```diff
@@ -2596,34 +2595,29 @@ cdef class Actor(Component):
         Raises
         ------
         ValueError
-            If `start` is > current timestamp (now).
+            If `start` is not `None` and > current timestamp (now).
         ValueError
-            If `end` is > current timestamp (now).
+            If `end` is not `None` and > current timestamp (now).
         ValueError
-            If `start` is > `end`.
+            If `start` and `end` are not `None` and `start` is >= `end`.
         TypeError
             If `callback` is not `None` and not of type `Callable`.
 
-        Notes
-        -----
-        When querying from a catalog, the engine will return the most recent instrument
```

Comment:
I agree that it's more intuitive that we don't apply this post processing by default. We should keep the default request as a basic `start` to `end` time range query, as a user may just want any _changes_ to instrument definitions and not necessarily always include the latest instrument prior to `start`.

Having said that, it _could_ be useful to be able to include these "prior to the query range active instruments", but I think we should add this functionality when necessary, and it should be controlled by an additional param and not be the default.